common/shared/src/test/java/app/coronawarn/server/common/shared/util/SecurityUtilsTest.java

```diff
@@ -47,6 +48,7 @@ void testEcdsaSignatureVerification()
     String signature = "+AE7sEXzNjgvxiDXrKdXQqL/XiOPIB/1r579jyIPWtQp7/a6K4m2vBsnjZSWvsZ+wT+WHkF8F64eCktNamZGhw==";
     byte[] base64DecodedSignature = base64decode(signature);
 
-    assertThrows(SignatureException.class, () -> ecdsaSignatureVerification(base64DecodedSignature, key, ""));
+    assertThrows(SignatureException.class,
```

Comment:
Can we maybe rename the test methods? I know we werent consistent in the past but I think a decriptive name helps other to grasp what the test is doing. Maybe something like shouldFailWhenSignatureDoesNotMatchContent. What do you think?